charts/astronomer/templates/houston/houston-configmap.yaml

```diff
@@ -71,6 +71,12 @@ data:
       kibanaUIEnabled: false
       {{ end }}
 
+      {{ if .Values.global.features.namespacePools.enabled }}
+      hardDeleteDeployment: true
+      manualNamespaceNames: true
+      preCreatedNamespaces:{{ printf "\n" }}{{ toYaml .Values.global.features.namespacePools.namespaces.names | indent 6 }}
+      {{ end }}
+
```

Comment:
If it's not enable don't we want to disable to make sure its not enabled within the service.

Reply:
@cmarteepants are we keeping manualReleaseNames and webhook configs at the service (houston) layer?

Reply:
We don't want to create breaking changes with this improvement.

So, if namespacePools is disabled, we just don't touch these houston parameters.

Also, the values inside .Values.astronomer.houston.config that you mentionned are still injected in a separate configuration file `local-production.yaml` which is more important (will take priority over the production.yaml created here), so users can still override this "default" configuration if they want to.